content/docs/intro/console/collaboration/project-and-stack-management.md

```diff
@@ -30,65 +30,65 @@ Every Pulumi program is deployed to a [stack]({{< relref "/docs/intro/concepts/s
+![Stack outputs and configuration](/images/docs/reference/service/stack-outputs-and-configuration.png)
 
 You can see other details such as who applied the update and when, as well as counts of added, updated, and unchanged resources. If your stack is integrated with your workflow, such as [GitHub Actions]({{< relref "/docs/guides/continuous-delivery/github-actions" >}}), you should also see useful links to data like your Git commit hash, mapped branch, and pull request ID.
 
 ##### Stack tags
 
-Custom [stack tags]({{< relref "/docs/intro/concepts/stack#stack-tags" >}}) can be managed within the Stack tab. Click the **NEW** button to create a new tag, the pencil to edit an existing tag, or the trash can to permanently delete a tag.
+Custom [stack tags]({{< relref "/docs/intro/concepts/stack#stack-tags" >}}) can be managed within the Stack tab. Select the **New** button to create a new tag, the pencil to edit an existing tag, or the trash can to permanently delete a tag.
 
-<img class="lg:max-w-xl" src="/images/docs/reference/service/stack-tags.png" alt="Stack Tags">
+![Stack tags](/images/docs/reference/service/stack-tags.png)
 
 #### Detailed Activity View
 
 By default, the Activity tab gives you a card-based list of stack updates sorted by date.
 
-Clicking on "Details" from the Stack update tab, or clicking on a specific activity from the login screen gives you a detailed view of that specific update. The Activity tab provides insights into the operations that were performed on your stack resources during an update. This tab can include three tabs, depending on your setup:
+Selecting "Details" from the Stack update tab, or selecting a specific activity from the login screen gives you a detailed view of that specific update. The Activity tab provides insights into the operations that were performed on your stack resources during an update. This tab can include three tabs, depending on your setup:
 
 * **Changes**. This lets you toggle between different log views:
-<img class="lg:max-w-xl m-6" src="/images/docs/reference/service/resource-changes.png" alt="Stack Resource Graph">
+![Stack resource graph](/images/docs/reference/service/resource-changes.png)
     * _Summary Log_ which lists a summary of changes, counts of affected resources, and update duration
     * _Diff Log_ which displays a diff of the changes (created, updated, or deleted resources), your stack outputs, and the same counts and update duration shown in the Summary Log view.
     * _Diagnostic Log_ which displays warning messages or a description of the operations performed during the update (if any).
 
 * **Timeline**. This gives you a detailed timeline of changes to individual cloud resources. It also includes useful resource links and counts of affected resources.
 
-<img class="lg:max-w-xl m-6" src="/images/docs/reference/service/timeline.png" alt="Stack Resource Graph">
+![Stack resource graph timeline](/images/docs/reference/service/timeline.png)
```

Comment:
Could we maybe update this image also, and bump the indendation a bit to line it up with the list item?

![image](https://user-images.githubusercontent.com/274700/112560997-8a009380-8d91-11eb-9b07-f8874844559d.png)

Reply:
ill try and do this tonight, but if not im good merging. as im going to redo all the screenshots right before 4/20 anyway.

Reply:
@cnunciato i updated 2 more screenshots on this page, i think that whole section needs to be re-wroked though, especially the hierarchy of information. but ive kinda let this pr snowball enough, so wont be taking it on in this pr.